packages/lib-react-components/src/ZooHeader/components/NarrowMenuNavListItem/NarrowMenuNavListItem.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { useTranslation } from 'react-i18next'
 
 import SpacedText from '../../../SpacedText'
 
-export default function NarrowMenuNavListItem ({ color, text }) {
+export default function NarrowMenuNavListItem ({ color, text, overrideLang }) {
+  const { i18n } = useTranslation()
   return (
     <SpacedText
       color={color}
-      weight='bold'
+      lang={overrideLang ? 'en' : i18n.language}
```

Comment:
Maybe add `lang` as an optional prop. That might be easier to read and understand.
```jsx
export default function NarrowMenuNavListItem({ color, text, lang }) {
…
  return (
    <SpacedText
      color={color}
      lang={lang}
      …
    />
```

Reply:
Can you clarify why `lang` should be considered optional here? If the `lang` prop is undefined, won't the `lang` attribute be set to "undefined"? I think that could be confusing in the html structure.

Reply:
Undefined values aren’t rendered by React, so `lang` would be ignored here by default.

Reply:
I see! Yes that does make more sense for readability.